exe/arduino_ci_remote.rb

```diff
@@ -228,7 +228,10 @@ def perform_compilation_tests(config)
   installed_library_path = attempt("Installing library under test") do
     @arduino_cmd.install_local_library(Pathname.new("."))
   end
-  if installed_library_path.exist?
+  if !installed_library_path
+    warning = "This probably indicates problems. Do you have a different/old library with the same name installed?"
+    inform("No libraries installed") { warning }
```

Comment:
This might produce better output if you use `inform_multiline("Library installation failed") { warning }`